website/docs/docs/dbt-versions/versionless-cloud.md

```diff
@@ -38,6 +48,26 @@ The legacy dbt Semantic Layer was deprecated in the second half of 2023. We reco
 
 </Expandable>
 
+<Expandable alt_header="What are other known issues when upgrading from older dbt Core versions?" >
+
+If you are upgrading from a very old unsupported version of dbt Core, you may run into one of these edge cases after the upgrade to a newer version:
+- [v1.3] Customers with non-dbt `.py` files defined within their project directories, such as `models/`. Since v1.3, dbt expects these files be valid [Python models](/docs/build/python-models). The customer needs to move these files out of their `models/` directory, or ignore them via `.dbtignore`
+- [v1.5] Customers who have `--m` in their job definitions, instead of `-m` or `--models`. This autocompletion" (`--m[odels]` for `--models`) has never been officially documented or supported. It was an implicit behavior of argparse (CLI library used in dbt-core v1.0-1.4) that is not supported by `click` (the CLI library used in dbt-core since v1.5+).
```

Comment:
```suggestion
- [v1.5] Customers who have `--m` in their job definitions, instead of `-m` or `--models`. This autocompletion (`--m[odels]` for `--models`) has never been officially documented or supported. It was an implicit behavior of argparse (CLI library used in dbt-core v1.0-1.4) that is not supported by `click` (the CLI library used in dbt-core since v1.5+).
```